package.json

```diff
@@ -44,7 +70,7 @@
     "@vue/cli-plugin-typescript": "^4.0.0",
     "@vue/cli-plugin-unit-jest": "^4.0.0",
     "@vue/cli-plugin-vuex": "^4.0.0",
-    "@vue/cli-service": "^4.0.0",
+    "@vue/cli-service": "^4.2.3",
```

Comment:
talvez seja isso q esteja atrapalhando o commit

Reply:
but my second commit didn't require to bypass, so let's just forget about it for now